filebeat/tests/system/test_modules.py

```diff
@@ -95,14 +97,21 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
         objects = [o["_source"] for o in res["hits"]["hits"]]
         assert len(objects) > 0
         for obj in objects:
-            self.assert_fields_are_documented(obj)
-            # assert "error" not in obj  # no parsing errors
-            assert obj["fileset"]["module"] == module
+            assert obj["fileset"]["module"] == module, "expected fileset.module={} but got {}".format(
+                module, obj["fileset"]["module"])
+
+            if not (module == "mysql" and fileset == "slowlog") and not (module == "system" and fileset == "auth"):
+                # TODO: There are errors parsing the test logs from these modules.
```

Comment:
I agree, definitely not idea, but I really wanted to enable `error` checking. Maybe as part of the manifest or some file in the test directory the fileset could self-describe its testing parameters (e.g. `allow_errors: true`, `has_dynamic_fields: true`).